hubblestack/extmods/modules/nebula_osquery.py

```diff
@@ -318,6 +319,53 @@ def get_top_data(topfile):
     return ret
 
 
+def mask_passwords(unmasked_object):
+
+    masked_object = copy.deepcopy(unmasked_object)
+
+    try:
+        mask_file  = __salt__['cp.cache_file']('salt://mask.yaml')
+        mask = yaml.safe_load(open(mask_file))
+        mask_by = mask.get('mask_by','******')
+        for r in masked_object:
+            for query_name, query_ret in r.iteritems():
+                if 'data' in query_ret:
+                    for result in query_ret['data']:
+                        for key, value in result.iteritems():
+                            if isinstance(value,basestring):
```

Comment:
Can you put a space after the comma? We don't have pylint checking live, but it will complain about this.